parcels/kernels/EOSseawaterproperties.py

```diff
@@ -0,0 +1,348 @@
+"""Collection of pre-built eos sea water property kernels"""
+import math
+
+__all__ = ['Pressure_from_LatDepth', 'AdiabticTemperatureGradient', 'Ptemp_from_Temp', 'Temp_from_Ptemp']
```

Comment:
Could you also remove the underscores int he names of the other inbuilt Kernels?